code/modules/meteors/meteors.dm

```diff
@@ -30,6 +30,8 @@ GLOBAL_LIST_INIT(meteorsD, list(/obj/effect/meteor/medium=15, /obj/effect/meteor
 //Meteor spawning global procs
 ///////////////////////////////
 
+//SKYRAT OVERRIDE START - overwritten in modular_skyrat\modules\events\code\directional_meteors.dm
+/*
 /proc/spawn_meteors(number = 10, list/meteortypes)
 	for(var/i in 1 to number)
 		spawn_meteor(meteortypes)
```

Comment:
Just make the one-line edits here, rather than copy-pasting the whole thing to the modular file.